core/templates/dev/head/editor/ExplorationHistory.js

```diff
@@ -53,6 +54,32 @@ oppia.controller('ExplorationHistory', [
     }
   });
 
+  $scope.submitVersion = function($event, versionNumber) {
+    var checkbox = $event.target;
+    var id = $scope.versionArr.indexOf(versionNumber);
+    if (checkbox.checked && id === -1) {
+      $scope.versionArr.push(versionNumber);
+    }
+    if (!checkbox.checked && id !== -1) {
+      $scope.versionArr.splice(id, 1);
+    }
+    
+    if($scope.versionArr.length == 2) {
+      $scope.versionCountPrompt = "";
+      $scope.changeCompareVersion(versionNumber);
+    }
+    else {
+      $scope.hideHistoryGraph = true;
+      $scope.hideCompareVersionsButton = false;
+
+      if($scope.versionArr.length == 0)
+        $scope.versionCountPrompt = "Please select any two.";
+      else if($scope.versionArr.length == 1)
+        $scope.versionCountPrompt = "Please select one more.";
+      else 
+        $scope.versionCountPrompt = "Please select exactly two.";      
```

Comment:
I think it would be better to follow the flow in Ben's mock -- i.e. disable the checkboxes. @virajprabhu, would it be possible to do this?